test_python/test_relax.py

```diff
@@ -24,4 +24,27 @@ def test_relax_keep_volume():
     params.iterations = 5
     res = mrmesh.relaxKeepVolume(keep_volume_torus, params)
 
-    assert (res)
\ No newline at end of file
+    assert res
+
+def test_relax_approx():
+    R1 = 2
+    R2_1 = 1
+    R2_2 = 2.5
+    torus = mrmesh.makeTorusWithSpikes(R1, R2_1, R2_2, 10, 12, None)
+
+    params = mrmesh.MeshApproxRelaxParams()
+    params.iterations = 5
+    res = mrmesh.relaxKeepVolume(torus, params)
+
+    assert res
+
+def test_smooth_region_boundary():
+    R1 = 2
+    R2_1 = 1
+    R2_2 = 2.5
+    keep_volume_torus = mrmesh.makeTorusWithSpikes(R1, R2_1, R2_2, 10, 12, None)
+
+    smooth_region = keep_volume_torus.topology.getValidFaces()
+
+    # This just checks that the function exists and can be called.
+    mrmesh.smoothRegionBoundary(keep_volume_torus, smooth_region)
```

Comment:
this function is not meant to be called with all valid faces, region with boudary required